components/measurement/StatusHeader.js

```diff
@@ -18,25 +18,24 @@ const StatusHeader = ({ status, color, icon, label, info }) => {
   let computedLabel = ''
   if (status) {
     switch (status) {
-      case 'anomaly':
-        computedLabel = 'Anomaly'
-        icon = <MdPriorityHigh />
-        break
-      case 'reachable':
-        computedLabel = 'Reachable'
-        icon = <MdCheck />
-        break
-      case 'error':
-        computedLabel = 'Error'
-        icon = <MdCheck />
-        break
-      case 'confirmed':
-        computedLabel = 'Blocking Confirmed'
-        icon = <MdCheck />
-        break
-      default:
-        computedLabel: ''
-        icon: <div />
+    case 'anomaly':
```

Comment:
This needs to be checked again. [Eslint documentation](https://eslint.org/docs/rules/indent#switchcase) says the earlier format is the correct one. Of course, this depends on what format we choose to follow.

Reply:
You mean the level of indentation?

My editor which should be using the eslint configuration (via `eslint --fix`) to autofix the fixable did that.

Running `yarn run lint` I see no errors in that particular file.

Do you use an editor plugin to auto-indent stuff and fix linting errors? Maybe we should just align our editor configs.

Reply:
I do use the same atom package. Looks like this is a side effect of some other tweak I made to my config elsewhere :man_facepalming: